package.json

```diff
@@ -40,6 +40,7 @@
     "tls-connect": "^0.2.2"
   },
   "optionalDependencies": {
+    "wrtc": "0.0.59",
```

Comment:
It's optional because it breaks the build on OSX. I'm hoping to fix that upstream, though.